src/utils/converter.ts

```diff
@@ -1,20 +1,32 @@
-import { utils } from 'near-api-js';
-import { Gas } from 'near-units';
-import { B_div_B, B_mul_N } from './math'
-
-
-const { formatNearAmount, parseNearAmount } = utils.format;
-
-const toSmall = (big: number | string, decimals: number): bigint => B_mul_N(BigInt("1" + "0".repeat(decimals)), big);
-const toLarge = (small: bigint, decimals: number): number => B_div_B(small, BigInt("1" + "0".repeat(decimals)))
+import { Big } from 'big.js';
+
+
+// config for Big.js behavior. see: https://mikemcl.github.io/big.js/
+Big.RM = 0;
+Big.DP = 40;
+Big.NE = -40;
+Big.PE = 40;
+
+const removeTrailingZeros = (amount: string): string => amount.replace(/\.?0*$/, '');
+// token amount -> indivisible units
+const parseTokenAmount = (amount: number | string, decimals: number): string => Big(amount).times(Big(10).pow(decimals)).toFixed();
```

Comment:
Don't wanna be that guy, but can we have more obvious naming? I'm gonna switch up `parse` and `format` all the time.
Also no need to have this work around with `toFixed()` and `removeTrailingZeros` (see [this line](https://github.com/near-multicall/ui/blob/535dc333f67fd5b1516fb1bbf5268659de0bf0e4/src/utils/converter.ts#L13) from #39)